api/framework/Core/FileStorage/Features/FileUploadCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MediatR;
+
+namespace FSH.Framework.Core.FileStorage.Features;
```

Comment:
can you put this under `Storage/File` folder, because at a later point, we would want to make the storage service to support Amazon S3 / Azure Blob Storage , etc. Also rename the interface to `IStorageService`

Reply:
Good idea. Done